client/components/empty-content/empty-content.jsx

```diff
@@ -32,12 +34,17 @@ module.exports = React.createClass( {
 		] ),
 		secondaryActionURL: React.PropTypes.string,
 		secondaryActionCallback: React.PropTypes.func,
+		onMount: React.PropTypes.func,
 		className: React.PropTypes.string,
 		isCompact: React.PropTypes.bool
 	},
 
 	componentDidMount: function() {
+		const { onMount } = this.props;
 		debug( 'Empty Content React component mounted.' );
+		if ( onMount ) {
+			onMount()
```

Comment:
We can alternatively render `lib/analytics/track-component-view` as a child in the render method instead of having a custom onMount.